server/controllers/stock/core.js

```diff
@@ -494,6 +475,59 @@ function getInventoryQuantityAndConsumption(params) {
     });
 }
 
+/**
+ * process multiple stock lots
+ *
+ * @description
+ * the goals of this function is to give the risk of peremption for each lots for
+ * a given inventory
+ */
+function processMultipleLots(inventories) {
+  const flattenLots = [];
+  const inventoryLots = _.groupBy(inventories, 'inventory_uuid');
+
+  _.each(inventoryLots, (inventory) => {
+    // order lots by ascending lifetime
+    const orderedInventoryLots = _.orderBy(inventory, 'lifetime', 'asc');
```

Comment:
Is there any way we can do the order by in SQL?  I worry this will be pretty slow.